core-web/apps/dotcms-ui/src/app/view/components/_common/dot-wizard/dot-wizard.component.spec.ts

```diff
@@ -309,31 +288,4 @@ describe('DotWizardComponent', () => {
             expect(formsContainer.nativeElement.style['transform']).toEqual('translateX(0px)');
         });
     });
-
-    describe('single step', () => {
```

Comment:
why removed ?

Reply:
The Enter key behavior doesn't work, and getting it to work interrupted with step change 
So I removed it